src/bundle/Resources/public/scss/_buttons.scss

```diff
@@ -25,7 +25,7 @@
         box-shadow: none;
         background-color: $ez-white;
         border-color: transparent;
-        color: #007DBE;
+        color: #131C26;
```

Comment:
why not in variable?

Reply:
Variable will be added later after full rebranding now we added by hardcode hex color